src/sidebar.css

```diff
@@ -25,6 +25,11 @@ body {
   display: flex;
   flex-direction: row;
   align-items: center;
+
+  user-select: none;
+  -ms-user-select: none;
+  -webkit-user-select: none;
```

Comment:
Haven't done any work elsewhere to port this to other browsers, so we can probably leave these out for now.